libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -310,122 +290,43 @@ - (void)enumerateDelegates:(void (^)(id<SFUserAccountManagerDelegate>))block
     }
 }
 
-#pragma mark - Temporary User
-
-- (SFUserAccount *)temporaryUser {
-    SFUserAccount *tempAccount = (self.userAccountMap)[self.temporaryUserIdentity];
-    if (tempAccount == nil) {
-        tempAccount = [self createUserAccount];
-    }
-    return tempAccount;
-}
-
 #pragma mark - Anonymous User
-
-+ (BOOL)isUserAnonymous:(SFUserAccount*)user {
-    if (nil == user.accountIdentity) {
-        return NO;
-    }
-    return [user.accountIdentity.userId isEqualToString:SFUserAccountManagerAnonymousUserAccountUserId] &&
-        [user.accountIdentity.orgId isEqualToString:SFUserAccountManagerAnonymousUserAccountOrgId];
-}
-
-- (BOOL)supportsAnonymousUser {
-    return [[[NSBundle mainBundle] objectForInfoDictionaryKey:kSFUserAccountSupportAnonymousUsage] boolValue];
-}
-
-- (BOOL)autocreateAnonymousUser {
-    return [[[NSBundle mainBundle] objectForInfoDictionaryKey:kSFUserAccountAutocreateAnonymousUser] boolValue];
-}
-
 - (BOOL)isCurrentUserAnonymous {
```

Comment:
We can move on from this I suppose, but I don't see this method as anything other than confusing.  It's a one-liner to replace an otherwise one-liner (`currentUser != nil`), so convenience doesn't really come into play.  And otherwise, there's no such thing, conceptual, as an "anonymous user".  There's no user that doesn't have credentials.